project-koku/koku
# [COST-4444] update processing to point to new model.

## Jira Ticket

[COST-4444](https://issues.redhat.com/browse/COST-4444)

## Description

This pull request changes the data processing over to the new model. 

## Testing

1. Load test customer data.
2. Make sure that the category=Platform functionality still works. 

## Notes

...
